Source/mpqapi.cpp

```diff
@@ -650,13 +613,12 @@ void mpqapi_flush_and_close(const char *pszArchive, BOOL bFree, int dwChar)
 }
 // 65AB0C: using guessed type int save_archive_modified;
 
-BOOLEAN mpqapi_write_header() // WriteMPQHeader
+BOOL mpqapi_write_header() // WriteMPQHeader
 {
-	BOOLEAN result;             // al
-	_FILEHEADER fhdr;           // [esp+8h] [ebp-6Ch]
-	DWORD NumberOfBytesWritten; // [esp+70h] [ebp-4h]
+	_FILEHEADER fhdr;
+	DWORD NumberOfBytesWritten;
 
-	memset(&fhdr, 0, 0x68u);
+	memset(&fhdr, 0, 0x68);

```

Comment:
Is `sizeof(_FILEHEADER)` 0x68 bytes? If so:

```suggestion
	memset(&fhdr, 0, sizeof(_FILEHEADER));
```